Scan/MatrixARM/matrix_scan.c

```diff
@@ -510,10 +534,6 @@ void Matrix_scan( uint16_t scanNum )
 			KeyPosition k = !st->cur
 				? (!st->prev ? KeyState_Off : KeyState_Release)
 				: ( st->prev ? KeyState_Hold : KeyState_Press);
-			//if (!st->cur && !st->prev)  k = KeyState_Off; else
```

Comment:
These 4 lines do exactly what the 2 lines above using ? and : so I though it's not needed.